test/e2e/auth/pod_security_policy.go

```diff
@@ -229,14 +228,14 @@ func testPrivilegedPods(tester func(pod *v1.Pod)) {
 	})
 }
 
-func createAndBindPSP(f *framework.Framework, pspTemplate *extensionsv1beta1.PodSecurityPolicy) (psp *extensionsv1beta1.PodSecurityPolicy, cleanup func()) {
+func createAndBindPSP(f *framework.Framework, pspTemplate *policy.PodSecurityPolicy) (psp *policy.PodSecurityPolicy, cleanup func()) {
```

Comment:
There is `createAndBindPSPInPolicy()` function (see below) that does the same. Just use it and remove this function.